src/modules/education/education.controller.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Patch,
+  Param,
+  Delete,
+} from '@nestjs/common';
+import { EducationService } from './education.service';
+import { CreateEducationDto } from './dto/create-education.dto';
+import { UpdateEducationDto } from './dto/update-education.dto';
+
+@Controller('education')
```

Comment:
hmm I dont think you need separate module for education, I think its better to move it to user, as you dont have separate page for this data, it always come together with user, get user data or put user data